modules/twinkleprod.js

```diff
@@ -355,57 +355,38 @@ Twinkle.prod.callbacks = {
 	},
 
 	addToLog: function(params) {
-		var wikipedia_page = new Morebits.wiki.page('User:' + mw.config.get('wgUserName') + '/' + Twinkle.getPref('prodLogPageName'), 'Adding entry to userspace log');
-		wikipedia_page.setCallbackParameters(params);
-		wikipedia_page.load(Twinkle.prod.callbacks.saveLog);
-	},
-
-	saveLog: function(pageobj) {
-		var text = pageobj.getPageText();
-		var params = pageobj.getCallbackParameters();
-
-		// add blurb if log page doesn't exist
-		if (!pageobj.exists()) {
-			text =
-				"This is a log of all [[WP:PROD|proposed deletion]] tags applied or endorsed by this user using [[WP:TW|Twinkle]]'s PROD module.\n\n" +
-				'If you no longer wish to keep this log, you can turn it off using the [[Wikipedia:Twinkle/Preferences|preferences panel]], and ' +
-				'nominate this page for speedy deletion under [[WP:CSD#U1|CSD U1]].\n';
-		}
+		var usl = new Morebits.userspaceLogger(Twinkle.getPref('prodLogPageName'));
+		usl.initialText =
+			"This is a log of all [[WP:PROD|proposed deletion]] tags applied or endorsed by this user using [[WP:TW|Twinkle]]'s PROD module.\n\n" +
+			'If you no longer wish to keep this log, you can turn it off using the [[Wikipedia:Twinkle/Preferences|preferences panel]], and ' +
+			'nominate this page for speedy deletion under [[WP:CSD#U1|CSD U1]].\n';
 
-		// create monthly header if it doesn't exist already
-		var date = new Date(pageobj.getLoadTime());
-		var headerRe = new RegExp('^==+\\s*' + date.getUTCMonthName() + '\\s+' + date.getUTCFullYear() + '\\s*==+', 'm');
-		if (!headerRe.exec(text)) {
-			text += '\n\n=== ' + date.getUTCMonthName() + ' ' + date.getUTCFullYear() + ' ===';
-		}
-
-		text += '\n# [[:' + Morebits.pageNameNorm + ']]';
+		var logText = '# [[:' + Morebits.pageNameNorm + ']]';
+		var summaryText;
```

Comment:
for ease of reviewing the diff, can this stay as `summarytext` rather than `summaryText`? (same with `text` vs `logText`, though the reason for changing that is clearer so its not as important to keep)

Reply:
Look at the diff with whitespace disabled. It's quite clear.